docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,34 +2,78 @@
 [role="xpack"]
 == SIEM UI
 
-The {siem-app} is a highly interactive workspace designed for security 
-analysts. It provides a clear, detailed overview of events from your 
-environment, and you can use the interactive UI to drill down into areas of 
+The {siem-app} is a highly interactive workspace designed for security
+analysts. It provides a clear, detailed overview of events from your
```

Comment:
nit: Perhaps the phrase `detailed overview` evokes a bit of contradiction here, because in general, overviews often obscure details to provide a "higher level" or summarized perspective of the data?

Reply:
@DonNateR 
This is mine :).
I agree with Andrew, let me know if you want me to deal with it. If not, I'll push a change to the PR.

Reply:
I went ahead and tweaked @benskelker